fuse-common.gemspec

```diff
@@ -29,13 +29,13 @@ Gem::Specification.new do |spec|
   spec.require_paths = %w[lib]
 
   # After updating Airbrake make sure that fuse_common/airbrake_libraries.rb is relevant
-  spec.add_dependency 'airbrake', '~> 9.0'
-  spec.add_dependency 'figaro', '~> 1.1'
-  spec.add_development_dependency 'bundler', '~> 1.16'
-  spec.add_development_dependency 'rails', '~> 4.0'
-  spec.add_development_dependency 'rake', '~> 10.0'
-  spec.add_development_dependency 'rspec', '~> 3.0'
-  spec.add_development_dependency 'rspec_junit_formatter', '~> 0.3'
+  spec.add_dependency 'airbrake', '~> 11.0.3'
+  spec.add_dependency 'figaro', '~> 1.2.0'
+  spec.add_development_dependency 'bundler', '~> 1.17.3'
+  spec.add_development_dependency 'rails', '~> 4.2.11.3'
```

Comment:
This will require use to make another change when we want to get to Rails 5 for the apps which have `fuse-common` as a dependency. That's not specifically a problem, but is something we will have to remember to address!

Since FuseTube doesn't have `fuse-common` as a dependency, we haven't encountered this yet.